run-tests.php

```diff
@@ -866,7 +866,7 @@ function write_information(array $user_tests, $phpdbg): void
         $exts = get_loaded_extensions();
         $ext_dir = ini_get('extension_dir');
         foreach (scandir($ext_dir) as $file) {
-            if (preg_match('/^(?:php_)?([_a-zA-Z0-9]+)\.(?:so|dll)$/', $file, $matches)) {
+            if (preg_match('/^(?:php_)([_a-zA-Z0-9]+)\.(?:so|dll)$/', $file, $matches)) {
```

Comment:
Note that this change is not directly related to this PR (although it is required to make this patch work on Windows, at least). If desired, I can provide a separate PR for this.

Reply:
Unfortunately I believe that the `php_` prefix is specific to windows (at least there is no prefix on linux).

Maybe using a block list would be fine. Otherwise I'm not sure how we can detect that a particular .so/.dll file is an extension. We could check if it defines a `${name}_module_entry` symbol but this will not be portable.

Reply:
> Unfortunately I believe that the `php_` prefix is specific to windows (at least there is no prefix on linux).

Ah, thank you! Maybe we should just used slightly different regexps on Windows and other platforms? Another option would be for Windows builds to put the extension DLLs in the ext/ folder; I'll have a look at this.

Reply:
Okay, I've pushed a commit separating the regexps (Windows vs. non-Windows). An early draft of putting the extension DLLs in a subfolder is PR #16282.